applications/HDF5Application/custom_io/hdf5_container_component_io.cpp

```diff
@@ -544,7 +544,7 @@ void ContainerComponentIO<TContainerType, TContainerItemType, TComponents...>::R
 
     if (mComponentNames.size() == 1 && mComponentNames[0] == "ALL_VARIABLES_FROM_FILE") {
         if (mpFile->HasPath(mComponentPath)) {
-            current_components_list = std::move(mpFile->GetDataSetNames(mComponentPath));
+            current_components_list = mpFile->GetDataSetNames(mComponentPath);
```

Comment:
~~~
In file included from /__w/Kratos/Kratos/build/Custom/applications/HDF5Application/CMakeFiles/KratosHDF5Core.dir/Unity/unity_0_cxx.cxx:15:
/__w/Kratos/Kratos/applications/HDF5Application/custom_io/hdf5_container_component_io.cpp:547:39: error: moving a temporary object prevents copy elision [-Werror,-Wpessimizing-move]
            current_components_list = std::move(mpFile->GetDataSetNames(mComponentPath));
                                      ^
/__w/Kratos/Kratos/applications/HDF5Application/custom_io/hdf5_container_component_io.cpp:547:39: note: remove std::move call here
            current_components_list = std::move(mpFile->GetDataSetNames(mComponentPath));
                                      ^~~~~~~~~~                                       ~
~~~

Reply:
on it :)

Reply:
it is already solved :)... missed that part.. .:)